### DISCUSSION TITLE
Can't create an account on foe-helper.com

### DISCUSSION THREAD

**Discussion by mainIine** [mainIine/foe-helfer-extension#2836] [General]
Add here your questions

**Comment by mainIine** [comment]
@p1d3c & @tbasnoopy
Both accounts are semi created. I deleted it. Pls register again.

After changing a routine to check all internal processes and tables it works now, I guess... 😄

**Reply by p1d3c** [reply]
Register success. When I'm trying to login I get an error "incorrect password", then I restored password and logged in. But profile page still show me this error

```
Warning: Undefined array key "current_world" in /usr/www/users/foecre/foe-helper.com/core/modul/user/pages/profile.php on line 112

Warning: Undefined array key "all_worlds" in /usr/www/users/foecre/foe-helper.com/core/modul/user/pages/profile.php on line 114

Fatal error: Uncaught TypeError: array_column(): Argument #1 ($array) must be of type array, null given in /usr/www/users/foecre/foe-helper.com/core/modul/user/pages/profile.php:114 Stack trace: #0 /usr/www/users/foecre/foe-helper.com/core/modul/user/pages/profile.php(114): array_column() #1 /usr/www/users/foecre/foe-helper.com/core/modul/user/user.php(34): include('/usr/www/users/...') #2 /usr/www/users/foecre/foe-helper.com/core/classes/class.layout.php(341): user->getTPL() #3 /usr/www/users/foecre/foe-helper.com/core/modul/user/user.php(28): layout->__construct() #4 /usr/www/users/foecre/foe-helper.com/index.php(50): user->__construct() #5 {main} thrown in /usr/www/users/foecre/foe-helper.com/core/modul/user/pages/profile.php on line 114
```

**Reply by tbasnoopy** [reply]
> Register success. When I'm trying to login I get an error "incorrect password", then I restored password and logged in. But profile page still show me this error

Same here

And on the api token site:
Warning: Undefined array key "de" in /usr/www/users/foecre/foe-helper.com/cache/smarty/templates_c/32e8f97d9014f3500921e1b26565b58f299c990b_0.file.profile.tpl.php on line 96

**Comment by Emm551** [comment]
I have the same problem.   I set up my account years ago.   I just got a new computer so I had to redo my password on my acct so that the basic FH tools appear on my new game screen.   However, I wanted to retrieve my memo tab data as well so I tried to do the key thing.  I only see the Mount Killmore sample code when I looked at that screen.  Then my FH Profile shows the above mentioned error msg when I look at my profile.  I really want this original account because lots of stuff is on my memo tab.  I have an eight digit  code which I think is my unique code but I somehow replaced that with the same error code as well.     I'd really appreciate any help you can give me on this.   I really don't want to create a new FH acct and lose that memo data.    Thank you

**Comment by mainIine** [comment]
I have built new queries in between. Try it again.

**Comment by craney3** [comment]
I am getting the follow error when trying to export my city.
when i go to my profile on foe helper i have this notification and I am not sure how I get this resolved - Undefined array key 1168 in /usr/www/users/foecre/foe-helper.com/cache/smarty/templates_c/32e8f97d9014f3500921e1b26565b58f299c990b_0.file.profile.tpl.php on line 98
and when i go to foehelper to get my API token, and I copy it and try to export my city, I get an error saying "This is not the correct api token"

**Comment by Bernini1** [comment]
Greetings,
I cannot login to FOEHelper site to access city planner. Main page displays;
Warning: Undefined array in key 5 in /usr/www/users/foecre/foe-helper.com/cache/smarty/templates_c/e22b74eb.....
My ingame name is Bernini. Can anyone help please? Thanks :)

**Comment by xfranz1** [comment]
Something is wrong with my user. Profile data is strange and can't get the API data to paste into the extension:
![Screenshot_3](https://github.com/user-attachments/assets/be42a498-50fd-4710-90c4-a4bee3c4f3a2)
If I click on tools -> city planner I get a very long error starting with: "Fatal error: Uncaught Error: Typed property citymap::$players_citydata must not be accessed before initialization in /usr/www/users/foecre/foe-helper.com/core/modul/citymap/citymap.php:254 ....... "
I tried to rerigester but the system tells me that a user with the same ID already exists. 
Can you please somehow reset my user?

**Comment by plejada** [comment]
Mam problem z zalogowaniem się na Hellera. Pokazuje sie info ze konta zostały usunięte po14 czerwca. Próbuje założyć nowe, pisze ze mail juz jest uzywany, jak przywracam hasło pojawia sie lista błędów. Na inny mail tez nie można założyć konta Co w takim wypadku?

**Comment by JanineElH** [comment]
Following this message: 'All accounts on this server were removed on 14th June 2025. Too many incorrect registrations were the reason, please register again. Existing notes for the extensions are automatically searched out and linked again.' I am trying to create a new account, but even after several tries and filling out everything required, it still fails.

**Comment by jfbinz-hub** [comment]
HI,
I have been trying to use City Planner in FOE for over a week now but I keep getting an error message that I have the wrong API token! I have copied and pasted many times from the site and checked the number and server over and over. What can I do to generate a new API token?>

**Comment by jason19999999999** [comment]
I can't create a account on FOE helper either... But in my case it's saying "Only one account per country is needed".... What the hell does that even mean?
